features/features.go

```diff
@@ -336,6 +336,14 @@ var (
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
+	FeatureGateAzureClusterHostedDNSInstall = newFeatureGate("AzureClusterHostedDNSInstall").
```

Comment:
Why does this one need the `*Install` suffix but the other platform gates did not?

Reply:
https://github.com/openshift/api/blob/master/tools/codegen/cmd/featuregate-test-analyzer.go#L566 requires names of Installer feature gates to contain "Install".
https://github.com/openshift/api/pull/2423 is starting the work of renaming existing featuregates.